java/meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/CommandManager.java

```diff
@@ -70,6 +70,12 @@ private static int getVersion() {
             }
         }
 
+        // Early-access releases add a "-ea" suffix
+        // These are the default versions for Alpine
+        if (version.endsWith("-ea")) {
```

Comment:
Looks like this is available as far back as at least jdk 1.0, seems good :+1:

https://javaalmanac.io/jdk/1.2/api/index.html